gemini/sample-apps/customer-search/functions/credit-card-imagen/main.py

```diff
@@ -0,0 +1,55 @@
+import base64
+import json
+from os import environ
+
+import functions_framework
+import vertexai
+from vertexai.preview.vision_models import ImageGenerationModel
+
+PROJECT_ID = environ.get("PROJECT_ID")
+LOCATION = environ.get("LOCATION")
+
+
+def get_prompt(request_json, request_args):
+    if request_json and "prompt" in request_json:
+        return request_json["prompt"]
+    elif request_args and "prompt" in request_args:
+        return request_args["prompt"]
+    else:
+        return "A dog reading the newspaper"
+
+
+def generate_base64_image(user_prompt):
+    vertexai.init(project=PROJECT_ID, location=LOCATION)
+    model = ImageGenerationModel.from_pretrained("imagegeneration@005")
+    images = model.generate_images(prompt=user_prompt, number_of_images=1, seed=1)
+
+    try:
+        images[0].save(location="./gen-img1.png", include_generation_parameters=True)
```

Comment:
You should be able to access the image base64 directly. https://github.com/googleapis/python-aiplatform/blob/118826ff88d6e0510dc67e37edfc16f646cb3489/vertexai/vision_models/_vision_models.py#L103C9-L103C21